client/app/components/PreviewCard.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// PreviewCard
+
+export function PreviewCard({ imageUrl, title, body, children, className, ...props }) {
+  return (
+    <div {...props} className={className + ' w-100 d-flex align-items-center'}>
+      <img src={imageUrl} height="32" className="profile__image--settings m-r-5" alt="Logo/Avatar" />
```

Comment:
Add `width="32"` or there will be a horizontal "jump" when image has loaded (happened to me).